policies-and-procedures/jellyfin-constitution.md

```diff
@@ -0,0 +1,155 @@
+
+2. Subproject Leaders are responsible for the day-to-day operations of their subproject and the handling of technical and managerial decisions for the subproject. For the purposes of all decisions pertaining to the subproject exclusively, the Subproject Leader shall be responsible for the final decision, in consultation with the Project Leader if necessary.
+
+3. Subproject Leaders shall have Admin privileges to the subproject repositories they manage and have the power to assign permissions on the subproject to other team members, for instance merge permissions, at their discretion.
+
+4. Subproject Leaders should seek consensus wherever possible with other contributors to the subproject, however they are empowered to make decisions as required. Disputes may be escalated to the Leadership Team or Project Leader if desired for further decision and resolution. As outlined in power #2, the Subproject Leader shall have the final say in any decision reached here.
+
+#### Procedure
+
+1. The initial Subproject Leader for a subproject shall be determined depending on the source of the subproject.
+
+     i. For subprojects that originate amongst the existing Jellyfin Project, for instance split-out libraries, or are forks of other projects, the subproject leader shall be determined by the Project Leader in consultation with the Leadership Team at the time the project is integrated into the Jellyfin Organization.
+
+    ii. For subprojects that originate as the sole work of an existing Jellyfin Team member, that person shall become Subproject Leader at the time the project is transferred to the Jellyfin Organization.
+
+   iii. For subprojects that originate as the sole work of an outside contributor who desires to make the project a subproject of the Jellyfin Organization, that person shall become a member of the Jellyfin Team, subject to standard nomination rules for Team Members, and shall become Subproject Leader at the time the project is transferred to the Jellyfin Organization. If nomination of the member fails for any reason, the project shall not be integrated into the Jellyfin Organization and shall remain a 3rd-party project.
+
+    iv. For subprojects that, at the time of the adoption of this consitution, do not already have a clear Subproject Leader, elections shall be held by the existing contributors to select the Subproject Leader. The method of election shall be ranked voting instant runoff until one candidate has secured an absolute majority of the vote.
+
+2. There shall always be exactly one Subproject Leader for any given project.
```

Comment:
How would this rule apply to the Android subproject specifically? Right now, both me and @nielsvanvelzen are admins of the GitHub repository, and we share the subproject leader role. I'd prefer to continue with that process, since managing the project and making decisions together has been very productive and I wouldn't even want to have the full authority myself.

Reply:
Very good point. Since you guys are already sharing it and it's already working out, and you both already have Admin, Co-Subproject Leaders (more like Andrew and I were in the early days) makes the most sense. You both have the powers but, ideally, come to consensus agreement on any action or decision. Does that work? /cc @nielsvanvelzen

Reply:
I have addressed this by allowing either one or two in this line. I actually didn't realize Android already had two :D

Reply:
How I see it is that I already am a subproject leader for android overall (so androidtv, android mobile and kotlin/java apiclient). @Maxr1998 is the creator of the current mobiel app and maintains most of it so I consider him as the subproject leader for the android mobile app underneath the "android" project.

Illustration because I find it hard to explain in text:

- Jellyfin
  - server/web
  - Android | Niels
    - jellyfin-android (mobile) | Max
    - jellyfin-androidtv | Niels - With assistance from Bill
    - jellyfin-apiclient-java | Niels
  - ...other things

Reply:
Thanks for the update abd the change, sounds good for me.

@nielsvanvelzen theoretically, I'd see the apiclient as independent from the Android subproject, and I'd even argue that (at least on paper) AndroidTV is an independent subproject from Android as well.
At least if we want to keep the hierarchy flat, because the structure you described sounds like having "sub-sub-projects".

Reply:
I think those three projects are related to each other so it's easy to "group" them, both apps use the apiclient so contributors for the apps might contribute to the apiclient etc.